.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
```

Comment:
I feel like this is implied in our process. No mergey mergey unless you get a +1. That's not 100% the case for all teams yet, I guess, but I _think_ that's our end goal, right?